packages/suite/src/services/github.ts

```diff
@@ -17,4 +19,49 @@ export const getReleaseNotes = async (version?: string) => {
     return release;
 };
 
+export const openGithubIssue = async () => {
+    const url = new URL(`${RELEASE_URL}/issues/new`);
+
+    let type = 'web';
+    if (isDesktop()) {
+        type = 'desktop app';
+    }
+
+    const body = `
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Steps to reproduce:**
+1. a
+2. b
+3. c
+
+**Info:**
+ - Suite Version: ${type} ${process.env.VERSION} (${process.env.COMMITHASH})
+ - Browser: ${getUserAgent()}
+ - Screen: ${getScreenWidth()}x${getScreenHeight()}
+ - OS: ${navigator.platform}
+ - Firmware Version: TODO
```

Comment:
would say device could be passed in params. In template you get it using `useDevice` hook

Reply:
Hm, I probably need help here. I have done exactly that but I am not sure what type I should declare the `device` argument in the function. See https://github.com/trezor/trezor-suite/commit/926c3ac637d1d3c9bb07f0d669b6ccb068cda346#diff-5348343ad9ec2fa9915061e16164cb286875912e9d43f5523f5f4d229bf02982R24. I've tried `TrezorDevice` but that doesn't build. What should I use?

Reply:
Maybe @slowbackspace or @keraf might help here? :angel: